README.md

```diff
@@ -1,34 +1,44 @@
-# GRASS Addons git repository
+# GRASS GIS Addons
 
-## How to get write access here
+This is the official GRASS GIS Addons git repository.
+Head over to [GRASS GIS 7Addons Manual
+pages](https://grass.osgeo.org/grass78/manuals/addons/)
+for an overview.
 
-Read access is granted to the public, write access
-must be requested, see here for details:
-<https://trac.osgeo.org/grass/wiki/HowToContribute#WriteaccesstotheGRASSaddonsrepository>
+## Install AddOns
 
-However, it is also possible to fork this repository, insert your AddOn or fix
-an existing one in a new branch and finally open
-a [pull request](https://help.github.com/en/articles/about-pull-requests).
+The simplest way to install GRASS GIS AddOns is to use the `g.extension`
+module which is part of any GRASS GIS installation:
 
-In either case, please read the submitting rules at the bottom of this page.
+<https://grass.osgeo.org/grass-stable/manuals/g.extension.html>
+
+The same module can remove installed addons
+
+## This repository
 
-## How to get the AddOn code
+### Clone locally
 
 Clone of the entire AddOns git repository:
 
 ```
 git clone https://github.com/OSGeo/grass-addons.git grass_addons
 ```
 
-## How to install or remove AddOns in your GRASS installation
+### Write access
 
-The simplest way to install GRASS GIS AddOns is to use the `g.extension`
-module:
-<https://grass.osgeo.org/grass-stable/manuals/g.extension.html>
+Read access is granted to the public, write access
+must be requested, see here for details:
+<https://trac.osgeo.org/grass/wiki/HowToContribute#WriteaccesstotheGRASSaddonsrepository>
+
+However, it is also possible to fork this repository, insert your AddOn or fix
```

Comment:
we should decide how to write addons: AddOns, add-ons, addons, etc?

Reply:
Was this ever decided elsewhere? For now, here it seems lowercase, one word, « addons » is used d’or talking about the repo, and « addon » for when talking for a single one.

Does this align with the user interface inside the software?

Reply:
Some more decision here https://github.com/OSGeo/grass-addons/pull/929